MetasysServices/MetasysClient.cs

```diff
@@ -512,10 +513,12 @@ public async Task<IEnumerable<MetasysObject>> GetObjectsAsync(Guid id, int level
                 parameters = new Dictionary<string, string>
                 {
                     { "depth", levels.ToString() },
-                    { "flatten", "true".ToString() }, //This parameter is needed to get the data in a 'flat' way and keep consistency in the logic to retrieve the objects
-                    { "includeExtensions", includeExtensions.ToString() },
-                    { "includeInternal", includeInternalObjects.ToString() } //This param has different name when version > v3
+                    { "flatten", "false" },
+                    { "includeExtensions", includeExtensions.ToString().ToLower() },
+                    { "includeInternal", includeInternalObjects.ToString().ToLower() } //This param has different name when version > v3

```

Comment:
These `toLower` method calls were added because the correct parameter values are `true` and `false`. I had to change so that my tests could pass.